README.md

```diff
@@ -15,7 +15,6 @@ To add a conference to the list, please [contribute](#contributing)!
 | [Deccan RubyConf 2018](https://www.deccanrubyconf.org/) | 4 Aug | Sheraton Grand Hotel, Pune | Pune extension of RubyConf India organized by same folks organizing RubyConf India |
 | [DroidJam](https://www.droidjam.in) | 13-14 July | Sheraton Grand, Bangalore | Conference on Android |
 | [Selenium Conf 2018](https://2018.seleniumconf.in/) | 28-30 June | Hotel Lalit Ashok, Bangalore  | Conference on Selenium Web Browser Automation tool |
-| [World Satoshi Summit](https://www.worldsatoshisummit.com/) | 12-13 May | New Delhi | Bring all the students, entrepreneurs, blockchain enthusiasts, blockchain developers and influencers under one roof for a seamless exchange of ideas. |
```

Comment:
Looking at the website, I think the event is still on?

Can you point me out to a reference where it says this event is cancelled? Thanks!

Reply:
> @nikhita Here you go: https://news.bitcoin.com/bitcoin-in-brief-friday-world-satoshi-summit-canceled-pump-and-dump-scheme-exposed/

Thanks, this does seem legit. Would have been better if they had updated their website as well. :)

I'll merge this because it definitely looks like the event is cancelled. In case anything changes or there is a new official announcement, please feel free to add it back.

Thanks for the clean up again, Amit! :)